README.md

```diff
@@ -211,6 +212,11 @@
 - [Full Web Apps with FastAPI Course](https://training.talkpython.fm/courses/full-html-web-applications-with-fastapi) - You'll learn to build full web apps with FastAPI, equivalent to what you can do with Flask or Django.
 - [The Definitive Guide to Celery and FastAPI](https://testdriven.io/courses/fastapi-celery/) - Learn how to add Celery to a FastAPI application to provide asynchronous task processing.
 
+### Books
+
+- [Building Data Science Applications with FastAPI](https://www.packtpub.com/product/building-data-science-applications-with-fastapi/9781801079211) - With this book, you will be able to create fast and reliable data science API backends using practical examples.
```

Comment:
I don't see any value in adding book links that are behind a paywall to this repo.

Reply:
Why not ?

Reply:
Almost every awesome list from https://github.com/sindresorhus/awesome I know has a books section. As long as all available books are included it's about showing the options available... not "advertising" someones book over another ones.